libs/pong/cc2ip.js

```diff
@@ -2,88 +2,180 @@ const cc2ip = {
   AD: "194.158.64.0",
   AE: "86.96.130.0",
   AF: "103.132.98.0",
+  AG: "66.249.150.51",
```

Comment:
In maxmind this shows up as Barbados rather than Antigua and Barbuda: makes me think we should perhaps have a test to ensure these country codes and IP addresses match: is this architecture likely to last?